### DISCUSSION TITLE
Feature request: Option to disable notifications / select frequency

### DISCUSSION THREAD

**Discussion by jonohewitt** [tosdr/browser-extensions#59] [Ideas] [answered]
I think an option to disable the notifications entirely, or perhaps select from a range of frequencies - e.g never, first site load, first load after rating change, monthly, weekly etc - would be really appreciated.

Personally I get the most use out of this extension by just looking at the letter grade in the URL bar and clicking it if interested. I'd much rather not have to deal with any of the notifications although I understand some people might find them valuable.

**Comment by JustinBack** [comment, marked as answer]
This functionality has now been added in a95c386b9f936a719f304a81f550a9ca0474a4ca. We will update the version on the webstore and mozilla addons accordingly.

The Notifications will show up by default however it can be turned off

![](https://9888-983528b48d7266437-14a87dc0-4ce4-11eb-b59c-5404a6a07802.imp.jbcdn.net/captures/firefox_NlXLRCfztH.png)

**Comment by jonohewitt** [comment]
Wow that was fast! Thanks a lot

**Comment by jonohewitt** [comment]
Hm I'm now getting this preferences panel in the Add-ons Manager page occasionally opening up by itself, with Notifcations re-selected, and taking the tab focus - using Firefox 85.0b9 (64-bit)

**Comment by JustinBack** [comment]
That should only happen on install of the plugin to initialize the settings.

Can you [file a bug report](https://github.com/tosdr/browser-extensions/issues/new/choose) for this?

**Reply by jonohewitt** [reply]
Sorry for the delay, just filed it.
